cmd/trivy.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+import (
+	"dynamic-buildkite-template/generator"
+)
+
+var (
+	trivyPluginConfig generator.TrivyPluginConfig
+	flagPrefix        = "trivy-"
```

Comment:
why can we not have a `trivyGenerateCmd` instead of all these flags? This will be incredibly confusing when its more than just trivy.

Reply:
done.